### ISSUE TITLE
Dropdown displays a list of [object object]

### ISSUE THREAD

**Issue by NicoleZA** [PeterStaev/NativeScript-Drop-Down#30]
If I use this code

`let dd = page.getViewById<DropDown>("dd");
let itemSource = new ValueList([{ ValueMember: "FL", DisplayMember:"Florida" }, { ValueMember: "MI", DisplayMember:"Michigan" }]);
dd.items = itemSource;`

then I get a list of objects and not a list of DisplayMembers

**Comment by PeterStaev** [comment]
Hey @NicoleZA , I've just tested again my source from the readme and it works on both platforms on the latest {N} version (2.2). Please make sure you copy the full code (including interface and class).

**Comment by toddanglin** [comment]
Just implemented using the demo code in this article and works for me, too. Also using {N} 2.2.

The first time I tried, I made the mistake of creating an Array of `ValueList` objects. So...

```
new Array<ValueList>();
```

For others following, check for this simple mistake if you're used to binding directly to an Array. To be clear, the CORRECT approach is:

```
let items = new Array<IValueItem>();
// Add items to array
let itemSource = new ValueList(items);
```
Hope that helps.
